src/models/article.ts

```diff
@@ -0,0 +1,146 @@
+import { PrismaClient, Prisma } from '@prisma/client';
+
+import { CreateData } from '../types/createArticle';
+import { UpdateData } from '../types/updateArticle';
+
+const prisma = new PrismaClient();
+
+export const createArticleData = async (data: CreateData) => {
+  const { title, category_id, content, image_id } = data;
+
+  try {
+    const newArticle = await prisma.article.create({
+      data: {
+        title,
+        category_id,
+        content,
+        image_id,
+      },
+    });
+
+    const filteredNewArticle = {
+      title: newArticle.title,
+      content: newArticle.content,
+      category_id: newArticle.category_id,
+      image_id: newArticle.image_id,
+      view: newArticle.view,
+    };
+    return { data: { ...filteredNewArticle } };
```

Comment:
This return should include success message

Reply:
On it.

Reply:
Done